woocommerce/woocommerce
# Rename classic template blocks to avoid confusion

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





The current name of classic template blocks may cause a confusion as they are pretty close to other blocks, especially `WooCommerce Single Product Block` vs `Single Product`.

In this PR I suggest renaming the classic templates block:
- from: `WooCommerce XYZ Block`
- to: `XYZ (Classic)`

so for example:

- `WooCommerce Product Attribute Block` -> `Product Attribute (Classic)`
- `WooCommerce Product Grid Block` -> `Product Grid (Classic)`

In addition:
- we simplified: `WooCommerce Single Product Block` -> `Product (Classic)`
- updated descriptions from generic one "Renders classic WooCommerce PHP templates." to more specific "Displays the PHP <<TITLE>> page." 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Editor -> Product Catalog
2. Search for "Classic Template"
3. Expected: There's only "Product Grid (Classic)" to choose
4. Add it and save
5. Go to frontend
6. Expected: Product Grid renders correctly

Repeat steps 3 - 6 for:
- "Product (Classic) Template" in Single Product template
- "Product Category (Classic)" in Products by Category template
- "Product Tag (Classic)" in Products by Tag template
- "Product Attribute (Classic)" in Products by Attribute template
- "Product Search Results (Classic)" in Product Search Results template



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
